### PR TITLE
Components

### PR DESCRIPTION
**Pull request by tsheporamantso**
## On this milestone:
- [ ] Created and used React components `MyCalculator.js` & `ChildResult.js` and imported them into top-level component `App.js
- [ ] Followed correct Gitflow
- [ ] Ensure there are no linter errors
- [ ] Followed JavaScript best practices

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,39 @@
-.App {
+.container {
+  display: flex;
+  flex-direction: column;
   text-align: center;
+  margin: 100px auto;
+  width: 300px;
+  padding: 2rem;
 }
```
Comment (OyePriscilla) [inline src/App.css:8]: - [ ] Please according to the requirement, kindly ensure that your design matches the given template; as developers, our ability to reproduce perfectly a given design shows how well we pay attention to details. ℹ️ 

> |     | Your Design                                  | Given Design                                |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | Untitled |Untitled |

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @tsheporamantso , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Implement React functional components ✔️
Remove all boilerplate that got created by the CRA ✔️

## Required Changes ♻️

- [x] Please kindly fix your live demo link, I was redirected to an empty page. **Note: You can remove the live demo header if there is no deployment link at the moment. Thanks** 🙏 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css (hunk 1 above)
Comment (tsheporamantso) [reply src/App.css:8]: Thank you so much @OyePriscilla, The equal sign button background color is not the same as the wireframe, I will fix it right away.

Comment (OyePriscilla) [reply src/App.css:8]: You welcome @tsheporamantso, I think the gap between each button should be reduced as well. 👍

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi @tsheporamantso 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**